newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -287,6 +289,7 @@ export const initialPreferences = {
     isAlwaysOnTopInPreview: false,
     backdropClickBehavior: 'nothing',
     eventsSheetCancelInlineParameter: 'apply',
+    eventsSheetUseAllExpressionTypes: false,
```

Comment:
I'm not sure I agree with the decision to make this prop false by default - meaning it's hardly an improvement from before.
For newcomers, it's really confusing/blocking the first time you use expressions.
Ex: I want to display the speed of an element in a text.
- if set to false, I don't see the `object.GetVelocityX()` autocompletion and I don't understand why, so I don't use it. (and maybe later I realise I can actually use it, but I have to know the expression by heart to type it)
- if set to true, I see the `object.getVelocityX()` autocompletion and I use it, then I get an error telling me I have to use toString() -> it teaches me that they are different types of variables and I have to understand the difference between number and string.

I definitely prefer the second option